presto-main/src/test/java/io/prestosql/server/TestDynamicFilterService.java

```diff
@@ -127,7 +127,7 @@ public void testDynamicFilterSummaryCompletion()
         assertEquals(
                 stats.getDynamicFilterDomainStats(),
                 ImmutableList.of(new DynamicFilterDomainStats(
-                        filterId, getExpectedDomainString(1L, 3L), 3, 0)));
+                        filterId, getExpectedDomainString(1L, 3L), 3, 0, Optional.empty())));
```

Comment:
fmt: all args to `new DynamicFilterDomainStats` should be on the `new DynamicFilterDomainStats` line, or each on separate line